etcd3/apis/kv.py

```diff
@@ -197,7 +197,9 @@ def range(
             "min_create_revision": min_create_revision,
             "max_create_revision": max_create_revision
         }
-        data = {k: v for k, v in data.items() if v is not None}
+        for k, v in list(data.items()):
```

Comment:
here a new list created, I'm not sure if it's better than just create a new dict.

Reply:
hah, I think I was typing out this exact thing when you were :)